### DISCUSSION TITLE
How to style admin login form in Headless mode

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#6972] [Q&A] [answered]
How to style admin login form in Headless mode

**Comment by Piedone** [comment]
What did you try?

**Comment by ns8482e** [comment]
@vitalybrandes `Login.cshtml` is be accessed by anonymous hence it uses front-end theme not admin theme.

It uses alternate layout `Layout__Login`.

**Comment by deanmarcussen** [comment, marked as answer]
I gave quite a detailed comment of the approach I am currently taking here which may help https://github.com/OrchardCMS/OrchardCore/issues/6877

If you are trying to style your admin login, in headless mode, then there is not much that could be done that is headless , because the admin is not part of that.

So to style the Admin Login you would need to forget about headless, and theme the login pages, via theme.

**Reply by vitalybrandes** [reply]
This was my question!
Thank you!
